gui/system/models.py

```diff
@@ -1231,58 +1238,82 @@ def is_enabled(self):
         return self.is_available(support=self)[0] and self.enabled
 
 
-class ACMERegistrationBody(models.Model):
+class ACMERegistrationBody(Model):
     contact = models.EmailField(
-        verbose_name='Contact Email'
+        verbose_name=_('Contact Email')
     )
     status = models.CharField(
-        verbose_name='Status',
+        verbose_name=_('Status'),
         max_length=10
     )
     key = models.TextField(
-        verbose_name='JWKRSAKey'
+        verbose_name=_('JWKRSAKey')
     )
     acme = models.ForeignKey(
         'ACMERegistration',
         on_delete=models.CASCADE,
     )
 
 
-class ACMERegistration(models.Model):
+class ACMERegistration(Model):
     uri = models.URLField(
-        verbose_name='URI'
+        verbose_name=_('URI')
     )
     directory = models.URLField(
-        verbose_name='Directory URI',
+        verbose_name=_('Directory URI'),
         unique=True
     )
     tos = models.URLField(
-        verbose_name='Terms of Service'
+        verbose_name=_('Terms of Service')
     )
     new_account_uri = models.URLField(
-        verbose_name='New Account Uri'
+        verbose_name=_('New Account Uri')
     )
     new_nonce_uri = models.URLField(
-        verbose_name='New Nonce Uri'
+        verbose_name=_('New Nonce Uri')
     )
     new_order_uri = models.URLField(
-        verbose_name='New Order Uri'
+        verbose_name=_('New Order Uri')
     )
     revoke_cert_uri = models.URLField(
-        verbose_name='Revoke Certificate Uri'
+        verbose_name=_('Revoke Certificate Uri')
     )
 
 
-class DNSAuthenticator(models.Model):
+class DNSAuthenticator(Model):
     authenticator = models.CharField(
         max_length=64,
+        verbose_name=_('Authenticator')
     )
     name = models.CharField(
         max_length=64,
-        unique=True
+        unique=True,
+        verbose_name=_('Name')
     )
     attributes = EncryptedDictField()
 
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        if self.attributes:
+            for key, value in self.attributes.items():
```

Comment:
`EncryptedDictField` stores encrypted JSON, can you please explain why is this necessary?